src/utils/fetch.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * This module exports an instance of fetch augmented with
+ * timeout and retries with exponential backoff.
+ */
+
+import { left, right } from 'fp-ts/lib/Either';
+import { fromEither, TaskEither } from 'fp-ts/lib/TaskEither';
+import { calculateExponentialBackoffInterval } from 'italia-ts-commons/lib/backoff';
+import { AbortableFetch, retriableFetch, setFetchTimeout, toFetch } from 'italia-ts-commons/lib/fetch';
+import { RetriableTask, TransientError, withRetries } from 'italia-ts-commons/lib/tasks';
+
+import { Millisecond } from 'italia-ts-commons/lib/units';
+import nodeFetch from 'node-fetch';
+
+// The old following import has been substituted by an explicit declaration
+// { fetchMaxRetries, fetchTimeout } from "../config";
+
+const fetchMaxRetries = 5;
```

Comment:
These values should be moved to a shared config file

Reply:
I agree.. I think this module should be refactored as a part of this story https://www.pivotaltracker.com/story/show/176339507, so I've done only the minimal interventions to decouple it from IO App and make the tests work